lib/src/bindings/6.5/generated_bindings.dart

```diff
@@ -180899,17 +180899,17 @@ abstract class csr_cs_core_usage_e {
   static const int CSR_CS_CORE_USAGE_SINGLE = 3;
 }
 
-class __csr_cs_context_s extends ffi.Opaque {}
+class _csr_cs_context_s extends ffi.Opaque {}
 
-class __csr_cs_malware_s extends ffi.Opaque {}
+class _csr_cs_malware_s extends ffi.Opaque {}
```

Comment:
Is this by design or a coincidence? If latter, you may check carefully.

Reply:
I removed multiple underscores.